diff --git a/Tables/dbo.Horoscope.sql b/Tables/dbo.Horoscope.sql
index be8d730..cfa34e7 100644
--- a/Tables/dbo.Horoscope.sql
+++ b/Tables/dbo.Horoscope.sql
@@ -1,6 +1,9 @@
 CREATE TABLE [dbo].[Horoscope]
 (
-[daily] [nvarchar] (1024) COLLATE Latin1_General_CI_AI NOT NULL,
+[ID] [int] NOT NULL IDENTITY(1, 1),
+[daily] [nvarchar] (1024) COLLATE Latin1_General_CI_AI NULL,
 [language] [nvarchar] (8) COLLATE Latin1_General_CI_AI NULL
 ) ON [PRIMARY]
 GO
+ALTER TABLE [dbo].[Horoscope] ADD CONSTRAINT [PK__Horoscop__3214EC2784448727] PRIMARY KEY CLUSTERED  ([ID]) ON [PRIMARY]
+GO
